orc8r/cloud/deploy/bare-metal-ansible/roles/magma/orc8r/files/orc8r-configs.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: orc8r-configs
+type: Opaque
+stringData:
+  metricsd.yml: |
+    profile: prometheus
+    prometheusQueryAddress: "http://orc8r-prometheus:9090"
+    alertmanagerApiURL: "http://orc8r-alertmanager:9093/api/v2"
+    prometheusConfigServiceURL: "http://orc8r-prometheus-configurer:9100"
```

Comment:
should be :
```
    prometheusConfigServiceURL: "http://orc8r-prometheus-configurer:9100/v1"
    alertmanagerConfigServiceURL: "http://orc8r-alertmanager-configurer:9101/v1"
```
according to the latest changes merged into master

Reply:
here's the source :
https://github.com/magma/magma/blob/1bed972e5a3c2f4fe31e650c7a3b94b8ecf8ad26/orc8r/cloud/configs/metricsd.yml